xarray/tests/test_indexing.py

```diff
@@ -824,14 +822,14 @@ def test_create_mask_outer_indexer() -> None:
 def test_create_mask_vectorized_indexer() -> None:
     indexer = indexing.VectorizedIndexer((np.array([0, -1, 2]), np.array([0, 1, -1])))
     expected = np.array([False, True, True])
-    actual = indexing.create_mask(indexer, (5,))
+    actual = indexing.create_mask(indexer, (5, 5))
```

Comment:
I think this is right. THe indexer is a 2-element tuple, so the array's shape should be 2D